src/Controller/AdminController.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+    /**
+     * @Route("/admin", name="admin_")
+     */
+class AdminController extends AbstractController
+{
+    /**
+     * @Route("/", name="index")
+     */
+    public function index(): Response
+    {
+        return $this->render('admin/index.html.twig', [
+            'controller_name' => 'AdminController',
```

Comment:
i think you forgot to delete that :slightly_smiling_face:

Reply:
done